### ISSUE TITLE
Travis failing because of incompatible version specs

### ISSUE THREAD

**Issue by dnowacki-usgs** [USGS-CMG/usgs-cmg-portal#315]
Travis pulls all the requirements.txt files, which are currently
`--file=./catalog_harvest/requirements.txt  --file=./usgs_rapid_release_gauges/requirements.txt  --file=./woods_hole_obs_data/requirements.txt  --file=./santa_cruz_obs_data/requirements.txt  --file=./whoi_obs_data/requirements.txt`

This results in the error:
```
UnsatisfiableError: The following specifications were found to be in conflict:
326  - netcdf4 <=1.2.8
327  - pyaxiom >=1.2.0 -> pynco -> nco -> esmf -> hdf5 >=1.10.4,<1.10.5.0a0
328  - pyaxiom >=1.2.0 -> pynco -> nco -> esmf -> libnetcdf >=4.6.2,<4.7.0a0
```
I'm guessing there are some out-of-date version specs in some of those requirements.txt that are no longer relevant...

**Comment by rsignell-usgs** [comment]
@dnowacki-usgs  , yeah, all that stuff should definitely be unpinned.   PR?

**Comment by dnowacki-usgs** [comment]
Closed by #316.
